workspaces/ui-v2/src/optic-components/pages/docs/DocumentationRootPage.tsx

```diff
@@ -87,7 +96,9 @@ export function DocumentationRootPage(props: {
                     disableGutters
                     style={{ display: 'flex' }}
                     onClick={() =>
-                      props.onEndpointClicked(endpoint.pathId, endpoint.method)
+                      history.push(
+                        `${match.url}/paths/${endpoint.pathId}/methods/${endpoint.method}`
```

Comment:
This handles both the documentation + changelog pages since it uses the current `match.url` - which is supplied from either the changelog page or the documentation page